tableau-scripts/tableau-backup.sh

```diff
@@ -41,7 +43,7 @@ elif (( $STAGING == 1 )); then
   export BACKUP_LOCATION="${DATA_ARCHIVE_TAB_BACKUP_URL}/staging/"
 else
   echo "== Set destination as Blue instance"
-  export BACKUP_LOCATION="${DATA_ARCHIVE_TAB_BACKUP_URL}/blue/"
+  export BACKUP_LOCATION="${DATA_ARCHIVE_TAB_BACKUP_URL}/${CURRENT_IP}/"
```

Comment:
I think it would be better to have:
`export BACKUP_LOCATION="${DATA_ARCHIVE_TAB_BACKUP_URL}/blue/${CURRENT_IP}/"`